<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>抗菌药物监测上报程序</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>700</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QPushButton" name="file_btn">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>210</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>加载Excel文件：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="file_path_text">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>451</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请选择待上报的Excel文件&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>请选择处方信息页：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>430</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>请选择药品信息页：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_btn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>580</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>开始填报</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>580</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>退出程序</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>选择上报类型：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="data_type">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>点击选择</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>门诊处方</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>急诊处方</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DDD</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>480</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>请选择 DDD信息页：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="presc_sheet">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sheet1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="drug_sheet">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>440</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sheet1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ddd_sheet">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>490</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Sheet1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sheet5</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>530</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>首页</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>530</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>550</width>
       <height>700</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,1,2,3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>楷体</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>处方上报页</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,20,10,44">
        <item>
         <widget class="QLabel" name="label_id">
          <property name="text">
           <string>处方ID：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="id"/>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>科室：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dep"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>姓名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>年龄：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="age">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>性别：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gender">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>男</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>女</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="spacing">
         <number>3</number>
        </property>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>诊断：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignBottom">
         <widget class="QTextEdit" name="diagnosis"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>用药：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="drug_info">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>宋体</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="rowCount">
           <number>5</number>
          </property>
          <property name="columnCount">
           <number>7</number>
          </property>
          <row/>
          <row/>
          <row/>
          <row/>
          <row/>
          <column>
           <property name="text">
            <string>药品名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>规格</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>用法</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>剂量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>频率</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>金额</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>进度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextBrowser" name="progress"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>530</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="text">
         <string>DDD上报页</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>530</width>
       <height>3</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>531</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>进度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="ddd_progress"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>531</width>
       <height>194</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>药品：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="ddd_drug_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <row/>
        <column>
         <property name="text">
          <string>药品名称</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>规格</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>用量</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>单价</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>金额</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
